pkg/operator/resource/resourcemerge/admissionregistration.go

```diff
@@ -0,0 +1,76 @@
+}
+
+// SetMutatingWebhooksConfigurationGeneration updates operator generation status list with last applied generation for provided MutatingWebhookConfiguration resource
+func SetMutatingWebhooksConfigurationGeneration(generations *[]operatorsv1.GenerationStatus, actual *admissionregistrationv1.MutatingWebhookConfiguration) {
+	if actual == nil {
+		return
+	}
+	SetGeneration(generations, operatorsv1.GenerationStatus{
+		Group:          admissionregistrationv1.SchemeGroupVersion.Group,
+		Resource:       "mutatingwebhookconfigurations",
+		Name:           actual.Name,
+		LastGeneration: actual.ObjectMeta.Generation,
+	})
+}
+
+// ExpectedValidatingWebhooksConfiguration returns last applied generation for ValidatingWebhookConfiguration resource registered in operator
+func ExpectedValidatingWebhooksConfiguration(name string, previousGenerations []operatorsv1.GenerationStatus) int64 {
+	generation := GenerationFor(previousGenerations, schema.GroupResource{Group: admissionregistrationv1.SchemeGroupVersion.Group, Resource: "validatingwebhookconfigurations"}, "", name)
+	if generation != nil {
+		return generation.LastGeneration
+	}
+	return -1
+}
+
+// SetValidatingWebhooksConfigurationGeneration updates operator generation status list with last applied generation for provided ValidatingWebhookConfiguration resource
+func SetValidatingWebhooksConfigurationGeneration(generations *[]operatorsv1.GenerationStatus, actual *admissionregistrationv1.ValidatingWebhookConfiguration) {
+	if actual == nil {
+		return
+	}
+	SetGeneration(generations, operatorsv1.GenerationStatus{
+		Group:          admissionregistrationv1.SchemeGroupVersion.Group,
+		Resource:       "validatingwebhookconfigurations",
+		Name:           actual.Name,
+		LastGeneration: actual.ObjectMeta.Generation,
+	})
+}
+
+// EnsureValidatingWebhooksConfiguration ensures that the existing matches the required.
+// modified is set to true when existing had to be updated with required.
+func EnsureValidatingWebhooksConfiguration(modified *bool, existing *admissionregistrationv1.ValidatingWebhookConfiguration, required admissionregistrationv1.ValidatingWebhookConfiguration) {
+	EnsureObjectMeta(modified, &existing.ObjectMeta, required.ObjectMeta)
+
+	// we stomp everything
+	if !equality.Semantic.DeepEqual(existing.Webhooks, required.Webhooks) {
+		*modified = true
+		existing.Webhooks = required.Webhooks
+	}
+}
+
+// EnsureMutatingWebhooksConfiguration ensures that the existing matches the required.
+// modified is set to true when existing had to be updated with required.
+func EnsureMutatingWebhooksConfiguration(modified *bool, existing *admissionregistrationv1.MutatingWebhookConfiguration, required admissionregistrationv1.MutatingWebhookConfiguration) {
+	EnsureObjectMeta(modified, &existing.ObjectMeta, required.ObjectMeta)
+
+	// we stomp everything
+	if !equality.Semantic.DeepEqual(existing.Webhooks, required.Webhooks) {
+		*modified = true
+		existing.Webhooks = required.Webhooks
+	}
```

Comment:
Is there any defaulting within the `Webhooks` that we need to account for here? We could end up in a hot loop if there's server side defaulting that we are missing